features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag

```diff
@@ -88,8 +88,26 @@ if (!user) {
         } else {
             //URL: https://localhost:9443/{deviceType}/download?owner={username}
             deviceManagerService = devicemgtProps["httpsURL"] + "/" + deviceType + "/manager";
-
             sketchDownloadEndPoint = deviceManagerService + "/device/" + sketchType + "/download";
+
+            //check if config exists
+            var deviceTypeConfigFile = new File("../app/units/cdmf.unit.device.type."+deviceType+".type-view/private/conf/device-type.json");
+            if(deviceTypeConfigFile.isExists()){
```

Comment:
this is already fixed with [#6c75515](https://github.com/wso2/carbon-device-mgt/commit/6c75515dc4d31e1c5f63845ee1d2f6a70bc1b156).